README.md

```diff
@@ -47,17 +47,13 @@ Examples of `Build` resource using the example strategies shipped with this oper
 
 ## Try it!
 
-- Install Tekton, optionally you could use
-[OpenShift Pipelines Community Operator][pipelinesoperator]
-
-- Install [operator-sdk][operatorsdk]
-
-- Create a project or namespace called **build-examples** by using `kubectl create namespace build-examples`
-
-- Execute `make local` to register [well-known build strategies](samples/buildstrategies) including **Kaniko**
-and start the operator.
-
-- Create a [Kaniko](samples/build/build_kaniko_cr.yaml) build
+* Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster, the supported version is v1.17.
+* Install Tekton by running [install-tekton.sh](hack/install-tekton.sh), the supported version is v0.11.3.
+* Install [operator-sdk][operatorsdk] by running [install-operator-sdk.sh](hack/install-operator-sdk.sh), the supported version v0.17.0
+* Create a namespace called **build-examples** by running `kubectl create namespace build-examples`
+* Execute `make local` to register [well-known build strategies](samples/buildstrategies) including **Kaniko**
```

Comment:
Do you want to point to the `samples/buildstrategy` folder? Your link url is wrong

Reply:
Good catch, the link was already wrong before my changes, I corrected it.